shop-auth-service/src/main/resources/db/migration/V1__Initial.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS `AUTH_DB`.`user` (
+  `user_id` BIGINT NOT NULL AUTO_INCREMENT,
+  `username` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(60) NOT NULL,
+  `is_account_non_expired` INT NOT NULL,
```

Comment:
```suggestion
  `is_account_non_expired` INT NOT NULL,
  `is_account_non_locked` INT NOT NULL,
  `is_credentials_non_expired` INT NOT NULL,
```

Nie było by lepiej odwrócić flagi? Znaczy, żeby wszystkie były domyślnie na `false` zamiast na `true`. 

Nie wiem jak reszta, ale dla mnie tak odwrócone zdają się być nieczytelne. W tym wypadku raczej bym chciał np. pisać zapytanie, że użytkownik jest zablokowany i dać `true`, a nie `false`.

Reply:
to wynika z tego że User implementuje org.springframework.security.core.userdetails.UserDetails
https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/core/userdetails/UserDetails.html

Reply:
Damn, no dobra. Z tym już nic nie zrobimy.